### ISSUE TITLE
Compilation error with Vaadin 24

### ISSUE THREAD

**Issue by javier-godoy** [FlowingCode/YearMonthCalendarAddon#24]
```
Module not found: Error: Can't resolve '@vaadin/vaadin-form-layout/src/vaadin-form-layout.js' 
in 'J:\fc\addons\fc-addons-demos-24\frontend\generated\jar-resources\fc-year-calendar'
```

**Comment by flang** [comment]
I wasn't able to fix it. I'm getting this error:

```
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker - [TypeScript] Found 0 errors. Watching for file changes.
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker - Failed to resolve import "@polymer/iron-a11y-keys-behavior/iron-a11y-keys-behavior.js" from "frontend\generated\jar-resources\fc-month-calendar\month-calendar-mixin.js". Does the file exist?
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker - Failed to resolve import "@polymer/iron-a11y-keys-behavior/iron-a11y-keys-behavior.js" from "frontend\generated\jar-resources\fc-month-calendar\month-calendar-mixin.js". Does the file exist?
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker - [vite] Internal server error: Failed to resolve import "@polymer/iron-a11y-keys-behavior/iron-a11y-keys-behavior.js" from "frontend\generated\jar-resources\fc-month-calendar\month-calendar-mixin.js". Does the file exist?
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker -   Plugin: vite:import-analysis
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker -   File: C:/dev/projects/Flowing code addons/YearMonthCalendarAddon/frontend/generated/jar-resources/fc-month-calendar/month-calendar-mixin.js:22:58
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker -   20 |  import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker -   21 |
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker -   22 |  import {IronA11yKeysBehavior} from '@polymer/iron-a11y-keys-behavior/iron-a11y-keys-behavior.js';
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker -      |                                                           ^
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker -   23 |
> [dev-server-output] INFO com.vaadin.base.devserver.DevServerOutputTracker -   24 |  import {ShadowFocusMixin} from '@vaadin/field-base/src/shadow-focus-mixin.js';
```

**Comment by javier-godoy** [comment]
I'll take a look.

**Comment by javier-godoy** [comment]
Closed in #25
